packages/wrangler/src/dev.ts

```diff
@@ -1071,177 +779,6 @@ export async function startDev(args: StartDevOptions) {
 	}
 }
 
-export async function startApiDev(args: StartDevOptions) {
```

Comment:
This has been entirely removed, since we no longer need a non-react equivalent of the dev flow—the normal dev flow is react-free!